src/com/vauff/maunzdiscord/commands/Services.java

```diff
@@ -295,6 +296,8 @@ else if (states.get(event.getAuthor().getStringID()).equals("csgoupdatesadd.1"))
 							states.remove(event.getAuthor().getStringID());
 							menuMessages.remove(event.getAuthor().getStringID());
 						}
+						
+						AbstractCommand.AWAITED.remove(event.getAuthor().getStringID()); //removing the author as he hasn't been removed because of line 287
```

Comment:
It's not a good idea to rely on that line number always being the same number as it is now.